src/utils/l1ToL2MessageToStatusDisplay.ts

```diff
@@ -5,33 +5,40 @@ import {
   MessageStatusDisplay,
 } from '@/types';
 import {
-  L1ToL2MessageReader,
   L1ToL2MessageStatus,
   L1ToL2MessageWaitResult,
 } from '@arbitrum/sdk/dist/lib/message/L1ToL2Message';
 import { looksLikeCallToInboxethDeposit } from './looksLikeCallToInboxethDeposit';
 
+async function getAutoRedeemAttempt(
+  l1ToL2Message:
+    | L1ToL2MessageReaderWithNetwork
+    | L1ToL2MessageReaderClassicWithNetwork,
+): Promise<string | undefined> {
+  if ('getAutoRedeemAttempt' in l1ToL2Message) {
+    return (await l1ToL2Message.getAutoRedeemAttempt())?.transactionHash;
+  }
+
+  return l1ToL2Message.autoRedeemId;
+}
+
 export const l1ToL2MessageToStatusDisplay = async (
   l1ToL2Message:
     | L1ToL2MessageReaderWithNetwork
     | L1ToL2MessageReaderClassicWithNetwork,
-  isClassic: boolean,
 ): Promise<MessageStatusDisplay> => {
   const { l2Network } = l1ToL2Message;
   const { explorerUrl } = l2Network;
 
   let messageStatus: L1ToL2MessageWaitResult | { status: L1ToL2MessageStatus };
+
   try {
-    if (isClassic) {
+    if ('waitForStatus' in l1ToL2Message) {
+      messageStatus = await l1ToL2Message.waitForStatus(undefined, 1);
```

Comment:
As a good practice, instead of `waitForStatus`, we should ideally use `getSuccessfulRedeem` method where we will not be needed to pass the arbitrary timeout param in the argument. Check out https://github.com/OffchainLabs/arbitrum-token-bridge/pull/745